### PR TITLE
Add docs for solver-parameters

### PR DESCRIPTION
**Pull request by tbittar**

### CODE REVIEW

File: docs/user-guide/solver/04-parameters.md [hunk 1]
```diff
@@ -472,13 +472,17 @@ _**This section is under construction**_
...
-[//]: # (TODO: document this parameter)
-_**This section is under construction**_  
 
-- **Expected value:**
-- **Required:** **yes**
-- **Default value:**
-- **Usage:**
+- **Expected value:** Any string
+- **Required:** no
+- **Default value:** `""`
```
Comment (pet-mit) [inline docs/user-guide/solver/04-parameters.md]:
```suggestion
- **Expected value:** any string
- **Required:** no
- **Default value:** empty string
```

File: docs/user-guide/solver/04-parameters.md [hunk 2]
```diff
@@ -472,13 +472,17 @@ _**This section is under construction**_
...
+- **Usage:** This parameter has effect only if:
+  - or-tools is used (argument `--use-ortools` from the command line) 
+  - solver `xpress` or `scip` is selected (using `--ortools-solver=<solver>` from the command line) as `sirius, glpk, coin` do not implment this functionality in or-tools. 
+  
+  The format is solver-specific and is the same as the corresponding solver configuration file format. This is an advanced usage and the user should therefore refer to the specific solver documentation to know how to specify the parameters. We give an example to set the maximum number of threads used during solve to 1:
+  - With Xpress : `solver-parameters = THREADS 1`
+  - With SCIP : `solver-parameters = parallel/maxnthreads = 1`
```
Comment (pet-mit) [inline docs/user-guide/solver/04-parameters.md]: can you add a second parameter in these examples please? (for example MAXNODE for xpress => `THREADS 1 MAXNODE 300000`, and its equivalent in SCIP, or any other parameter)
to allow the users to see the syntax to use to define multiple parameters in this single string (eq in xpress parameters are sperated by an empty space)

**Review by pet-mit — CHANGES_REQUESTED** [review]

**Comment by pet-mit** [comment]
also, this doesn't render well (see here: https://antares-simulator--2042.org.readthedocs.build/en/2042/user-guide/solver/04-parameters/#solver-parameters)

- for nested lists in mkdocs, you must have 2 extra empty spaces before the `-` of the nested list
- sometimes you have to add an empty space before a list in order for it to render well
- the "The format is solver-specifi..." paragraph should be in the "usage" part

you can test locally by using the `mkdocs serve` command (see here for some tips: https://github.com/AntaresSimulatorTeam/Antares_Simulator/blob/doc/optim_update/docs/README.md)

**Comment by tbittar** [comment]
Yes indeed, I was using the rendering preview of md file from VScode but it is different from the mkdocs page

**Review by pet-mit — APPROVED** [review]

File: docs/user-guide/solver/04-parameters.md [hunk 3]
```diff
@@ -472,13 +472,19 @@ _**This section is under construction**_
...
+- **Default value:** empty string
+- **Usage:** This parameter has effect only if:
+    - or-tools is used (argument `--use-ortools` from the command line) 
+    - solver `xpress` or `scip` is selected (using `--ortools-solver=<solver>` from the command line) as `sirius, glpk, coin` do not implment this functionality in or-tools. 
+  
+    The format is solver-specific and is the same as the corresponding solver configuration file format. This is an advanced usage and the user should therefore refer to the specific solver documentation to know how to specify the parameters. We give an example to set the maximum number of threads used during solve to 1 and to activate the presolve:
+      
+      
+      - With Xpress : `solver-parameters = THREADS 1 PRESOLVE 1`
+      - With SCIP : `solver-parameters = parallel/maxnthreads 1, lp/presolving TRUE`
```
Comment (flomnes) [inline docs/user-guide/solver/04-parameters.md]:
```suggestion
      - With Xpress : `xpress-parameters = THREADS 1 PRESOLVE 1`
      - With SCIP : `scip-parameters = parallel/maxnthreads 1, lp/presolving TRUE`
```

**Review by flomnes — CHANGES_REQUESTED** [review]
Where is the user supposed to specify these new parameters ?

File: docs/user-guide/solver/04-parameters.md (hunk 3 above)
Comment (tbittar) [reply docs/user-guide/solver/04-parameters.md]: All parameters within this file relate to `generaldata.ini`, see description at beginning of the file

**Review by pet-mit — APPROVED** [review]

File: docs/user-guide/solver/04-parameters.md (hunk 3 above)
Comment (pet-mit) [reply docs/user-guide/solver/04-parameters.md]: OK to merge @flomnes ?

**Comment by tbittar** [comment]
This PR is now useless given the choice that have been made to only set solver parameters from command line, should we close it @pet-mit @flomnes ?

**Comment by tbittar** [comment]
Instead I opened #2100

**Pull request closed without merging**